setup.py

```diff
@@ -18,23 +18,15 @@
 
 here = os.path.abspath(os.path.dirname(__file__))
 
-about = {}
-with open(os.path.join(here, 'simple_salesforce', '__version__.py'), 'r') as f:
-    exec(f.read(), about)
-
 
 setup(
-    name=about['__title__'],
```

Comment:
Remove your fork specific details

Reply:
sorry my fault. I have to get a version of this running on a project for work. The latest commit was meant for another repo. Should be fixed now